wfe2/wfe.go

```diff
@@ -2214,6 +2336,8 @@ func (wfe *WebFrontEndImpl) NewOrder(
 	order, err := wfe.ra.NewOrder(ctx, &rapb.NewOrderRequest{
 		RegistrationID: acct.ID,
 		Names:          names,
+		Replaces:       replaces,
```

Comment:
When `replaces` is an empty string, that's fine because this won't technically be a replacement order?

Reply:
Yep, if you go look at the RA code you'll see that we only take an action if it's a non-empty string.